plugins/modules/cloudfront_response_headers_policy.py

```diff
@@ -0,0 +1,282 @@
+#!/usr/bin/python
+# Copyright (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+version_added: 3.0.1
```

Comment:
For now this would be 3.1.0 but the release date is approaching fast (14/02/22).

Reply:
yes, this code still needs some polishing and would probably no get ready in time.